commons/src/main/java/com/powsybl/commons/extensions/AbstractExtensionAdder.java

```diff
@@ -19,7 +19,7 @@
 public abstract class AbstractExtensionAdder<T extends Extendable<T>, E extends Extension<T>>
         implements ExtensionAdder<T, E> {
 
-    private final T extendable;
+    protected final T extendable;
```

Comment:
It seems you do that for `RemoteReactivePowerAdderImpl`. I'd suggest to keep this private and make `RemoteReactivePowerAdderImpl` implement `Validable` with
```java


    @Override
    public String getMessageHeader() {
        return "generatorRemoteReactivePowerControl of generator " + extendable.getId();
    }
```

and use this `ValidationUtil.checkReactivePowerControl(this, regulatingTerminal, targetQ, enabled, extendable.getNetwork());`

Also a note in `RemoteReactivePowerAdderImpl`: do the checks in `createExtension` method, not before

Reply:
Done. But in the method `getMessagerHeader `i put a generic message as I do not have access to `extendable`.